compiler/il/AliasSetInterface.hpp

```diff
@@ -256,7 +256,7 @@ class TR_AliasSetInterface {
    setAliases_impl(TR::SparseBitVector &aliasesToModify, bool value, bool isDirectCall, bool includeGCSafePoint)
       {
       TR::Compilation *comp = TR::comp();
-      TR_ASSERT_FATAL(_AliasSetInterface == UseDefAliasSet || _AliasSetInterface ==  UseOnlyAliasSet, "failed: can only call setAliases_impl for UseOnly or useDef presently");
+      TR_STATIC_ASSERT(_AliasSetInterface == UseDefAliasSet || _AliasSetInterface ==  UseOnlyAliasSet, "failed: can only call setAliases_impl for UseOnly or useDef presently");
```

Comment:
So, I don't think `TR_STATIC_ASSERT` is defined anywhere. Using a regular C++ `static_assert` should work though (https://en.cppreference.com/w/cpp/language/static_assert).

Reply:
Sorry I've already done a commit with that changes